KR2d0006_146.txt

```diff
@@ -1,1176 +1,1913 @@
+博士齊王攸之就國也下禮官議崇錫之物旉與博士¶
+太叔廣劉暾繆蔚秦秀傅珍等上表諫曰書稱帝堯克¶
+明俊德以親九族武王光有天下兄弟之國十有六人¶
+<pb:KR2d0006_WYG_146-100a>¶
+同姓之國四十人元勲睦親顯以殊禮而魯衞齊晉大¶
+啓土宇並受分器所謂惟善所在親疎一也大晉龍興¶
+隆唐周之遠迹王室親屬佐命功臣咸受爵土而四海¶
+乂安今吳㑹已平詔大司馬齊王出統方岳當遂撫其¶
+國家將凖古典以垂永制昔周之選建明德以左右王¶
+室也則周公為太宰康叔為司冦聃季為司空又召芮¶
+畢毛諸國皆入居公卿大夫之位明股肱之任重守地¶
+之位輕也未聞古典以三事之重出之國者漢氏諸侯¶
+<pb:KR2d0006_WYG_146-100b>¶
+王位尊勢重在丞相三公上其入讚朝政者乃有兼官¶
+其出之國亦不復假台司虛名為隆寵也昔申無宇曰¶
+五大不在邊先儒以為貴寵公子公孫累世正卿也又¶
+曰五細不在庭先儒以為賤妨貴少陵長遠間親新間¶
+舊小加大也不在庭不在朝庭為政也又曰親不在外¶
+羈不在内今棄疾在外鄭丹在内君其少戒之叔向有¶
+言公室將卑其枝葉先落公族公室之本而去之諺所¶
+謂芘焉而縱尋斧柯者也今使齊王賢邪則不宜以母¶
+<pb:KR2d0006_WYG_146-101a>¶
+弟之親尊居魯衞之常職不賢邪不宜大啓土宇表建¶
+東海也古禮三公無職坐而論道不聞以方任嬰之惟¶
+周室大壞宣王中興四夷交侵救急朝夕然後命召穆¶
+公征淮夷故其詩曰徐方不囘王曰旋歸宰相不得乆¶
+在外也今天下已定六合為家將數延三事與論太平¶
+之基而更出之去王城二千里違舊章矣旉草議先以¶
+呈父純純不禁太常鄭黙博士祭酒曹志並過其事武¶
+帝以博士不答所問答所不問大怒事下有司尚書朱¶
+<pb:KR2d0006_WYG_146-101b>¶
+整褚䂮等奏旉等侵官離局迷罔朝廷崇飾惡言假託¶
+無諱請收旉等八人付廷尉科罪旉父純詣廷尉自首¶
+旉以議草見示愚淺聽之詔免純罪廷尉劉頌又奏旉¶
+等大不敬棄市論求平議尚書又奏請報聽廷尉行刑¶
+尚書夏侯駿謂朱整曰國家乃欲誅諫臣官立八座正¶
+為此時卿可共駮正之整不從駿怒起曰非所望也乃¶
+獨為駁議左僕射魏舒右僕射下邳王晃等從駿議奏¶
+留中七日乃詔曰旉等備為儒官不念奉憲制不指答¶
+<pb:KR2d0006_WYG_146-102a>¶
+所問敢肆其誣罔之言以干亂視聽而旉是議主應為¶
+戮首但旉及家人並自首大信不可奪秦秀傅珍前者¶
+虛妄幸而得免復不以為懼當加罪戮以彰凶慝猶復¶
+不忍皆丐其死命秀珍旉等並除名後數歳復起為散¶
+騎常侍&KR0008;於國子祭酒¶
+秦秀字元良新興雲中人也父朗魏驍騎將軍秀少敦¶
+學行以忠直知名咸寧中為博士何曾卒下禮官議諡¶
+秀議曰故太宰何曾雖階世族之允而少以高亮嚴肅¶
+<pb:KR2d0006_WYG_146-102b>¶
+顯登王朝事親有色養之名在官奏科尹模此二者實¶
+得臣子事上之槩然資性驕奢不循軌則詩云節彼南¶
+山維石巖巖赫赫師尹民具爾瞻言其德行高峻動必¶
+以禮耳邱明有言儉德之恭侈惡之大也大晉受命勞¶
+謙隱約曾受寵二代顯赫累世曁乎耳順之年身兼三¶
+公之位食大國之租荷保傅之貴執司徒之均二子皆¶
+金貂卿校列於帝側方之古人責深負重雖舉門盡死¶
+猶不稱位而乃驕奢過度名被九域行不履道而享位¶
+<pb:KR2d0006_WYG_146-103a>¶
+非常以古義言之非惟失輔相之宜違斷金之利也穢¶
+皇代之美壞人倫之敎生天下之醜示後世之慠莫大¶
+於此自近世以來宰相輔臣未有受垢辱之聲被有司¶
+之劾父子塵累而䝉恩貸若曾者也周公弔二季之陵¶
+遲哀大敎之不行於是作諡以紀其終曾參奉之啓手¶
+歸全易簀而没蓋明愼終死而後已齊之史氏亂世陪¶
+臣耳猶書君賊累死不懲况於皇代守典之官敢畏彊¶
+盛而不盡禮管子有言禮義亷耻是謂四維四維不張¶
+<pb:KR2d0006_WYG_146-103b>¶
+國乃滅亡宰相大臣人之表儀若生極其情死又無貶¶
+是則帝室無正刑也王公貴人復何所畏哉所謂四維¶
+復何寄乎謹按諡法名與實爽曰繆怙亂肆行曰醜曾¶
+之行已皆與此同宜諡繆醜公時雖不同秀議而聞者¶
+懼焉秀性忌讒佞疾之如讐素輕鄙賈充及伐吳之役¶
+聞其為大都督謂所親曰充文案小才乃居伐國大任¶
+吾將哭以送師或止秀曰蹇叔知秦軍必敗故哭送其¶
+子耳今吳君無道國有自亡之形羣率踐境將不戰而¶
+<pb:KR2d0006_WYG_146-104a>¶
+潰子之哭也旣為不智乃不赦之罪於是遂止及孫皓¶
+降於王濬充未之知方以吳未可平抗表請班師充表¶
+與告㨗同至朝野以充位居人上智出人下僉以秀為¶
+知言及充薨秀議曰充舍宗族弗授而以異姓為後悖¶
+禮溺情以亂大倫昔鄫養外孫莒公子為後春秋書莒¶
+人滅鄫聖人豈不知外孫親邪但以義推之則無父子¶
+耳又按詔書自非功如太宰始封後如太宰所取必已¶
+自出如太宰不得以為比然則以外孫為後自非元功¶
+<pb:KR2d0006_WYG_146-104b>¶
+顯德不之得也天子之禮蓋可然乎絶父祖之血食開¶
+朝廷之禍門諡法昬亂紀度曰荒請諡荒公不從王濬¶
+有平吳之勲而為王渾所譖毁帝雖不從無明賞罰以¶
+濬為輔國大將軍天下咸為之怨秀乃上言曰自大晉¶
+啓祚輔國之號率以舊恩此為王濬無功之時受九列¶
+之顯位立功之後更得寵人之辱號也四海視之孰不¶
+失望蜀小吳大平蜀之後二將皆就加三事今濬還而¶
+降等天下安得不惑乎吳之未亡也雖以三祖之神武¶
+<pb:KR2d0006_WYG_146-104a>¶
+潰子之哭也旣為不智乃不赦之罪於是遂止及孫皓¶
+降於王濬充未之知方以吳未可平抗表請班師充表¶
+與告㨗同至朝野以充位居人上智出人下僉以秀為¶
+知言及充薨秀議曰充舍宗族弗授而以異姓為後悖¶
+禮溺情以亂大倫昔鄫養外孫莒公子為後春秋書莒¶
+人滅鄫聖人豈不知外孫親邪但以義推之則無父子¶
+耳又按詔書自非功如太宰始封後如太宰所取必已¶
+自出如太宰不得以為比然則以外孫為後自非元功¶
+<pb:KR2d0006_WYG_146-105a>¶
+顯德不之得也天子之禮蓋可然乎絶父祖之血食開¶
+朝廷之禍門諡法昬亂紀度曰荒請諡荒公不從王濬¶
+有平吳之勲而為王渾所譖毁帝雖不從無明賞罰以¶
+濬為輔國大將軍天下咸為之怨秀乃上言曰自大晉¶
+啓祚輔國之號率以舊恩此為王濬無功之時受九列¶
+之顯位立功之後更得寵人之辱號也四海視之孰不¶
+失望蜀小吳大平蜀之後二將皆就加三事今濬還而¶
+降等天下安得不惑乎吳之未亡也雖以三祖之神武¶
+<pb:KR2d0006_WYG_146-105a>¶
+猶躬受其屈以孫皓之虛名足以驚動諸夏每一小出¶
+雖聖心知其垂亡然中國輙懷惶怖當爾時有能借天¶
+子百萬之衆平而有之與國家結兄弟之交臣恐朝野¶
+實皆甘之耳今濬舉蜀漢之卒數旬而平吳雖舉吳人¶
+之財寶以與之本非已分有焉而遽與計較乎後與劉¶
+暾等同議齊王攸事忤旨除名尋復起為博士秀性悻¶
+直與物多忤為博士前後垂二十年卒於官¶
+¶
+<pb:KR2d0006_WYG_146-105b>¶
 　¶
 　¶
 　¶
 　¶
 　¶
-　通志卷一百二十四下¶
+　¶
+　通志卷一百二十三¶
```

Comment:
The text here is provided by myself, as previously they were missing from the `.txt`:

https://github.com/kanripo/KR2d0006/blob/d1445a47c0910abcee4c31bb383081a0249dc375/KR2d0006_144.txt#L1870-L1878

Please proofread against the source images: https://img.kanripo.org/general/skqs/wyg/WYG0377/WYG0377-0586c.png and https://img.kanripo.org/general/skqs/wyg/WYG0377/WYG0377-0586d.png. Thank you.

Reply:
Sorry, I thought this was bundled in the other PR.  Now applying this as well.  And thanks for adding the missing text.